src/main/scala/mesosphere/marathon/core/matcher/base/util/ActorOfferMatcher.scala

```diff
@@ -29,13 +29,6 @@ class ActorOfferMatcher(actorRef: ActorRef, override val precedenceFor: Option[P
 
   override def toString: String = s"ActorOfferMatcher($actorRef)"
 
-  override def isInterestedIn(offer: Offer) = {
```

Comment:
Where did this method go?

Reply:
It's true by default, and we need exactly this

Reply:
If it's `true` in the trait and we never override it - do we really need it?